bofire/domain/objectives.py

```diff
@@ -18,11 +19,27 @@
 TWeight = confloat(gt=0, le=1)
 
 
+class BotorchConstrainedObjective:
+    """This abstract class offers a convenience routine for transforming sigmoid based objectives to botorch output constraints."""
+
+    @abstractmethod
+    def get_callables(self, idx: int) -> List[Callable[[Tensor], Tensor]]:
```

Comment:
maybe rename to `get_constrained_objectives`

Reply:
What do you think about `get_constraint_callables`? I want to make clear that this method transforms the objective to a botorch output constraint that is expressed via a callable.

Reply:
The fact that it returns callables is already documented by the return type.

Reply:
This a very valid argument :D, then just `get_constraints`?

Reply:
Or maybe `to_constraints` since this is more a conversion of the object than a property getter?

Reply:
This is perfect. I will change it accordingly ;)

Reply:
I renamed it.